Formula/run.rb

```diff
@@ -0,0 +1,22 @@
+class Run < Formula
+  desc "Easily manage and invoke small scripts and wrappers"
+  homepage "https://github.com/TekWizely"
```

Comment:
```suggestion
  homepage "https://github.com/TekWizely/run"
```

`homepage` should point to the homepage for the product, not for the author.

Reply:
Thanks man, missed that.